### PR TITLE
Hidden Security Section of the Docs

### PR DESCRIPTION
**Pull request by mandrigin**

### CODE REVIEW

**Comment by tersec** [comment]
A couple of initial comments:

(1) This seems, quite appropriately, to focus on security as such, contra privacy, though they to an extant can share analogous approaches. Still, it may be worth considering how to extend this discussion to generalize to privacy aspects which don't strictly align with the quite OWASP-centric framework being utilized here.

(2) Still, despite (1), I agree that it's worth prioritizing security, given more easily achieved consensus, even at Status, that security's a goal worth achieving unto itself, without which functionality's not as valuable (in various literal and metaphorical senses of the word).

**Comment by tersec** [comment]
"This is potentially a one-time project. The goal is to identify threats in our existing features, risk-assess them and document them in GHI." depends on stable threat models, as well. In general, the maxim/cliché that security is a process, rather than primarily a state or outcome, tends to be well-grounded. Still, I'm not sure what the right balance between creating discrete, achieveable steps, and a more sustainable security culture (re OWASP, for example), lies here.

I'm also wondering if it's worth mentioning GDPR apropos PII, or whether that's already so widely known (and understood, beyond as a headline buzzword) it's not worthwhile. The purpose would be emphasizing that PII handling has legal bite, not simply "should be"-sorts of best practices. For the most part, these don't currently apply to Status as much as more data-slurping organizations, as far as I can tell, but as e.g., ENS ramps up, with the sometimes-identity-verification, that sort of thing will become more acute.

**Comment by tersec** [comment]
How do you see this as addressing app-centric, per se, aspects, versus sort of domain (Ethereum, public key crypto, etc) security issues? Status being in a particularly conceptually challenging area (crypto, either in the blockchain or cryptography sense) suggests that https://people.eecs.berkeley.edu/~tygar/papers/Why_Johnny_Cant_Encrypt/OReilly.pdf https://cups.cs.cmu.edu/soups/2006/posters/sheng-poster_abstract.pdf https://arxiv.org/abs/1510.08555 etc (which document how email encryption has been apparently unusable broadly for 20+ years due to what the presenter at the status security meetup would call "use errors" orthogonally to app[lication] design errors) means that addressing "security" might exceed bounds of OWASP-type "attack surface" approaches to an extent.

File: website/sidebars.json [hunk 1]
```diff
@@ -2,6 +2,7 @@
   "docs": {
     "Introduction": ["introduction", "faqs"],
     "Build Status": ["build_status", "contributing"],
+    "Security": ["security_experiment", "security_process", "security_existing_features", "security_new_features", "security_champions", "security_github_labels", "security_sensitive_data", "security_threat_modeling"],
```
Comment (jeluard) [inline website/sidebars.json]: Can you remove this for now? We'd rather have it in an iteration after MVP.

**Comment by mandrigin** [comment]
@tersec 
>> "This is potentially a one-time project. The goal is to identify threats in our existing features, risk-assess them and document them in GHI."
>  In general, the maxim/cliché that security is a process, rather than primarily a state or outcome, tends to be well-grounded.

That is written for the backlog of existing features only, retroactive threat modelling. For the new features, updates and code changes security should be a process.

**Comment by mandrigin** [comment]
@jeluard I removed it from the sidebar, btw

**Comment by mandrigin** [comment]
@tersec I have updated the documents according to your feedback, please take a look one more time. If you have any better sources of information, articles, examples, feel free to share them.

**Pull request merged**